Dashboards/README.md

```diff
@@ -1,11 +1,13 @@
-## Sample Dashboards
+## Community submitted dashboards
 
-This folder contains sample dashboards from the community that can be used to get a quickstart on a new dashboarding project.
+Find inspiration and real-life dashboard use cases submitted by the SquaredUp Commmunity.
+
+Browse these folders to explore the sample dashboards that can be used to get a quickstart on a new dashboarding project.
 
 You'll find a mix of complete dashboards with loads of awesome examples or just simple views to share a single tile.
 
 Note the folder strucutre of `PRODUCT_TYPE_TITLE` when browsing.
 
 ### How to use these samples
 
-Some info text goes here... AK to flesh out.
\ No newline at end of file
+Each folder will contain readme documenation to guide you through the process of setting up the dashboard for yourself.
```

Comment:
I was planning to add some stuff here that applies to all i.e. a couple of steps like "add a new dashboard, open the JSON editor, paste over the top etc"... and for people wanting to contribute... some tips like stripping signing info, checking it has no keys embedded in queries etc....

Reply:
A readme in each dashboard for how to "install" the dashboard is still required though, yes repetitive, it will mean than when someone goes from the gallery to the folder the instructions for that specific dashboard will be shown. And the contribution guidelines I will cover in the top level folder.  

WDYT?

Reply:
But if someone gives the wrong instructions on how to do it that inevitably becomes a support issue. I'd rather we had the basics covered at least once "officially" in there at the top level.

Keep in mind the file you're asking change said "AK to flesh out".... feel free to leave this to me :)

Reply:
OK sounds good, I'll leave it with you 👍